priv/emqx.schema

```diff
@@ -426,12 +583,18 @@ end}.
   {datatype, file}
 ]}.
 
-{mapping, "sasl", "sasl.sasl_error_logger", [
+{mapping, "log.sasl_error_logger", "sasl.sasl_error_logger", [
   {default, off},
   {datatype, flag},
   hidden
 ]}.
 
+{mapping, "log.error_logger", "kernel.error_logger", [
+  {default, silent},
+  {datatype, {enum, [silent]}},
```

Comment:
what does this on-symbol enum mean? does it mean `silent or not` ?
isn't `true | false` enum more clear than a `undefined | silent` ?